UI/data/themes/Dark.qss

```diff
@@ -142,7 +142,14 @@ QDockWidget::close-button:pressed, QDockWidget::float-button:pressed {
 QGroupBox {
     border: 1px solid rgb(31,30,31); /* veryDark */;
     border-radius: 5px;
-    padding-top: 16px;
+    padding-top: 24px;
+    font-weight: bold;
```

Comment:
Should the font weight be moved to the QGroupBox::title css?

Reply:
I tried that and it didn't work.

Reply:
Some kind of explanation was here: https://github.com/obsproject/obs-studio/pull/1610#issuecomment-457090528